src/api-client.ts

```diff
@@ -57,9 +57,25 @@ export class RevAiApiClient {
             { 'Content-Type': 'application/json' }, 'json', options);
     }
 
-    async submitJobLocalFile(filename: string, options?: RevAiJobOptions): Promise<RevAiApiJob> {
+    async submitJobFileStream(
+        fileStream: Buffer | Readable,
+        filename: string,
+        options?: RevAiJobOptions
+    ): Promise<RevAiApiJob> {
         let payload = new FormData();
-        payload.append('media', fs.createReadStream(filename));
+        console.log(fileStream);
+        payload.append('media', fileStream, { filename: filename });
+        if (options) {
+            payload.append('options', JSON.stringify(options));
+        }
+
+        return await this.apiHandler.makeApiRequest<RevAiApiJob>('post', `/jobs`,
+            payload.getHeaders(), 'json', payload);
+    }
+
+    async submitJobLocalFile(filepath: string, options?: RevAiJobOptions): Promise<RevAiApiJob> {
```

Comment:
Unrelated to the PR, but i noticed there is no doc comments for any of the methods, is that deliberate?

in the python SDK we have doc comments - it would seem nice to have them for node as well

Reply:
Agreed, I think it's just been an oversight. The streaming-client was more recently written and does have doc comments. I'll file an issue